include/tester/db/Database.hpp

```diff
@@ -93,27 +66,14 @@ class Database
         return bsoncxx::stdx::nullopt;
     }
 
-
-
-    bsoncxx::stdx::optional<bsoncxx::document::value> getLog(std::string collectionName, std::string id)
+    bsoncxx::stdx::optional<bsoncxx::document::value> getDocument(std::string collectionName, std::string id)
     {
-        mongocxx::collection collection = _database[collectionName];
+        mongocxx::collection collection = _database[std::move(collectionName)];
         return collection.find_one(bsoncxx::builder::stream::document{} 
             << "_id" << bsoncxx::oid{id} << bsoncxx::builder::stream::finalize);
```

Comment:
`id` is a parameter passed by value. use `std::move`

Reply:
done